Week1/exercise2.js

```diff
@@ -0,0 +1,71 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    database: 'new_world'
+});
+
+connection.connect();
+
+connection.query('SELECT Name,Population FROM country WHERE Population > 8000000', function (error, results, fields) {
+    if (error) throw error;
+    console.log(`===========================================================`);
+    console.log(`Countries with population over 8M`);
+    results.forEach(item => console.log(item.Name, item.Population));
+});
+connection.query('SELECT Name FROM country WHERE Name LIKE "%land"', function (error, results, fields) {
```

Comment:
IMO "Countries that have 'land' in their names" doesn't necessarily mean "ends in -land" as you have implemented, so it should've been `%land%`.